velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -871,11 +883,12 @@ void DuckDbQueryRunner::createTable(
           auto value = ::duckdb::Value::INTERVAL(
```

Comment:
@mbasmanova : At the time I had followed changes for IntervalDayTime() as logical type. So maybe we need to change ::duckdb::Value duckValueAt<TypeKind::BIGINT> for IntervalDayTime as well?

Reply:
@aditi-pandit This is a good point. Would you help make this change?

Reply:
Sure. Thanks.